package.json

```diff
@@ -79,7 +86,10 @@
     "@fortawesome/free-solid-svg-icons": "~5.15.1",
     "@ideditor/temaki": "~5.1.0",
     "@mapbox/maki": "^6.0.0",
+    "@playwright/test": "^1.21.1",
     "autoprefixer": "^10.4.7",
+    "babel-plugin-inline-json-import": "^0.3.2",
```

Comment:
I've scanned the diffs but I can't see where this is used?  It's kind of suspicious, since we did recently get rid of all the Babel from the project and targeting modern browsers now.